samples/client/petstore/python/docs/FakeApi.md

```diff
@@ -175,55 +173,7 @@ configuration = petstore_api.Configuration(
 with petstore_api.ApiClient() as api_client:
     # Create an instance of the API class
     api_instance = fake_api.FakeApi(api_client)
-    xml_item = XmlItem(
-        attribute_string="string",
-        attribute_number=1.234,
-        attribute_integer=-2,
-        attribute_boolean=True,
-        wrapped_array=[
-            1,
-        ],
-        name_string="string",
-        name_number=1.234,
-        name_integer=-2,
-        name_boolean=True,
-        name_array=[
-            1,
-        ],
-        name_wrapped_array=[
-            1,
-        ],
-        prefix_string="string",
-        prefix_number=1.234,
-        prefix_integer=-2,
-        prefix_boolean=True,
-        prefix_array=[
-            1,
-        ],
-        prefix_wrapped_array=[
-            1,
-        ],
-        namespace_string="string",
-        namespace_number=1.234,
-        namespace_integer=-2,
-        namespace_boolean=True,
-        namespace_array=[
-            1,
-        ],
-        namespace_wrapped_array=[
-            1,
-        ],
-        prefix_ns_string="string",
-        prefix_ns_number=1.234,
-        prefix_ns_integer=-2,
-        prefix_ns_boolean=True,
-        prefix_ns_array=[
-            1,
-        ],
-        prefix_ns_wrapped_array=[
-            1,
-        ],
-    ) # XmlItem | XmlItem Body
+    xml_item = XmlItem(XmlItem) # XmlItem | XmlItem Body
```

Comment:
Your update is breaking these samples.
- Client
- AnimalFarm
- XmlItem

Can you update your code so these work correctly?